frontend/mgramseva/lib/utils/constants.dart

```diff
@@ -57,11 +57,11 @@ class Constants {
       Routes.HOUSEHOLD_REGISTER,
       {},
     ),
-    HomeItem(
+/*    HomeItem(
```

Comment:
Remove unwanted comments

Reply:
this commented because of we want to hide this but in futute we need this module so i just comment that module